Dalamud/Plugin/Services/IGameConfig.cs

```diff
@@ -31,6 +37,16 @@ public interface IGameConfig
     /// </summary>
     public event EventHandler<ConfigChangeEvent> UiControlChanged; 
 
+    /// <summary>
+    /// Gets a task representing the initialization state of this instance of <see cref="IGameConfig"/>.
+    /// </summary>
+    /// <remarks>
+    /// Accessing <see cref="GameConfigSection"/>-typed properties such as <see cref="System"/>, directly or indirectly
+    /// via <see cref="TryGet(Game.Config.SystemConfigOption,out bool)"/>,
+    /// <see cref="Set(Game.Config.SystemConfigOption,bool)"/>, or alike will block, if this task is incomplete.
+    /// </remarks>
+    public Task InitializationTask { get; }
```

Comment:
What's the intended use for this Task? 

That is, under what conditions would a plugin want to read or care about this, and can we not just block service completion until GameConfig is ready? (What's the downside to doing this?)

Reply:
I was thinking about the case where a plugin might want to queue setting retrieval upon plugin constructor using `.ContinueWith` to that task, but `IFramework.RunOnTick` can be used instead. It's been removed.